src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -284,7 +284,7 @@ public void AsymmetricRSAMixedCaseAttributesVerifyWindows()
             Assert.True(v1.CheckSignature());
         }
 
-        [Fact(Skip = "https://github.com/dotnet/corefx/issues/16691")]
+        [Fact]
         public void AsymmetricDSASignature()
```

Comment:
Gotcha - so really it's more a case of (if there _was_ such a method, which there isn't because one is normally created when an `AsymmetricAlgorithm` is created) the method for generating a new key throwing something like `PlatformNotSupportedException`?